boards/common/nrf52/include/cfg_timer_default.h

```diff
@@ -42,11 +42,29 @@ static const timer_conf_t timer_config[] = {
         .channels = 3,
         .bitmode  = TIMER_BITMODE_BITMODE_08Bit,
         .irqn     = TIMER2_IRQn
+    },
+    {
+        .dev      = NRF_TIMER3,
```

Comment:
NRF_TIMER3 is not available on `nrf52805`, `nrf52811` &  `nrf52810` either I believe.